lib/rules/no-unused-vars.js

```diff
@@ -18,6 +18,8 @@ const astUtils = require("../util/ast-utils");
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I'd consider `no-unused-vars` to be a `problem`; a lot of errors that it reports are a bugs that would result from forgetting to use a particular variable.

Reply:
I agree, let's see what others think.

Reply:
I agree this should probably be a "problem".

Reply:
Setting to "problem"